data/json/items/gunmod/accessories.json

```diff
@@ -18,6 +18,25 @@
     "dispersion_modifier": -20,
     "min_skills": [ [ "gun", 1 ] ]
   },
+  {
+    "id": "arrowrest_wood",
+    "type": "GUNMOD",
+    "name": { "str": "arrow rest" },
```

Comment:
Shouldn't this have a different name 🤔 "wooden arrow rest"

Reply:
In connection with #61629, can you take it *off* once you put it on?

(I assume this is what you did by "I've done it myself!")

Reply:
A different name didn't seem necessary, but if it's preferable I can change it no problem.

I put the new arrowrest on the primitive bows and took it off without issue in testing the PRs together. Good idea.

When I say I've done it myself I mean I've put a wooden arrowrest onto a primitive bow with cordage before. If I can do it myself I figure it should be doable in the game, too.